clients/ui/frontend/src/__mocks__/mockModelRegistryService.ts

```diff
@@ -0,0 +1,71 @@
+export type ServiceKind = {
+  kind: 'Service';
+  apiVersion: string;
+  metadata: {
+    name: string;
+    namespace: string;
+    annotations: {
+      'openshift.io/description': string;
+      'openshift.io/display-name': string;
+      'routing.opendatahub.io/external-address-rest': string;
+    };
+  };
+  spec: {
+    selector: {
+      app: string;
+      component: string;
+    };
+    ports: {
+      name: string;
+      protocol: string;
+      port: number;
+      targetPort: number;
+    }[];
+  };
+};
+
+type MockServiceType = {
+  name?: string;
+  namespace?: string;
+  description?: string;
+  serverUrl?: string;
+};
+
+export const mockModelRegistryService = ({
```

Comment:
Just curious to know, any reason we created this - I see that we have not used this mock anywhere.